client/src/components/App/App.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import styled, { css } from 'styled-components';
 import Reviews from 'Reviews/Reviews.jsx';
 import Overview from 'Overview/Overview.jsx';
+import RelatedItemsList from '../RelatedItems/RelatedItemsList.jsx'
```

Comment:
We have aliases set up to avoid having to navigate directories. You can just import from `'RelatedItems/RelatedItemsList.jsx`.